go/vt/vttablet/tabletserver/querythrottler/file_based_config_loader_test.go

```diff
@@ -176,9 +167,9 @@ func TestFileBasedConfigLoader_Load(t *testing.T) {
 			config, err := loader.Load(context.Background())
 
 			// Assert
-			if tt.expectedErrorNotNil {
+			if tt.expectedError != "" {
 				require.Error(t, err)
```

Comment:
nitpick: since you are doing `require.EqualError`, you don't need `require.Error`